ste/jobStatusManager.go

```diff
@@ -128,6 +128,9 @@ func (jm *jobMgr) handleStatusUpdateMessage() {
 			}
 			js.CompleteJobOrdered = js.CompleteJobOrdered || msg.IsFinalPart
 			js.TotalTransfers += msg.TotalTransfers
+			if js.TotalTransfers > common.RECOMMENDED_OBJECTS_COUNT {
```

Comment:
I dont think we need to add this here - just the one in ExecuteNewCopyJobPartOrder is sufficient. 

What scenario does this execute in that the above doesnt?